src/index.md

```diff
@@ -192,10 +192,10 @@ Mesh2_face_edges:long_name = "Maps every triangular face to its three edges." ;
 Mesh2_face_edges:start_index = 1 ;
 integer Mesh2_face_links(nMesh2_face, Three) ;
 Mesh2_face_links:cf_role = "face_face_connectivity" ;
-Mesh2_face_links:long_name = "Indicates which other faces neighbor each face." ;
+Mesh2_face_links:long_name = "neighbor faces for faces" ;
 Mesh2_face_links:start_index = 1 ;
-Mesh2_face_links:flag_values = -1 ;
-Mesh2_face_links:flag_meanings = "out_of_mesh" ;
+Mesh2_face_links:_FillValue = -999 ;
```

Comment:
This is correct, of course, but I think the example should use -1 as the _FillValue -- that's in common usage.